apps/todos/src/app/core/directives/index.ts

```diff
@@ -0,0 +1 @@
+export * from './directives.module';
```

Comment:
you don't need to add an index file only for the module, from my side I really don't see an advantage for this case (with modules), please remove everywhere from application in the same cases

Reply:
Thanks, I did it